### PR TITLE
[8.7][CSPM] Adds CSPM overview and getting started pages to Cloud Security section

### PR DESCRIPTION
**Pull request by benironside**
Addresses #2788 by adding new pages for the CSPM integration, and editing some formerly KSPM-only docs to apply to both the KSPM and CSPM integrations. Creates a new subsection of the top-level section "Cloud native security" called "Security posture management" which contains all CSPM and KSPM docs.

Preview (explore the whole Security posture management section):
[CSPM Overview
](https://security-docs_3014.docs-preview.app.elstc.co/guide/en/security/master/cspm.html)

### CODE REVIEW

File: docs/cloud-native-security/cspm.asciidoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[[cspm]]
+= Cloud security posture management
+
+The Cloud Security Posture Management (CSPM) integration discovers and evaluates the services in your cloud environment, like storage, compute, IAM, and more, against configuration security guidelines defined by the https://www.cisecurity.org/[Center for Internet Security] (CIS) to help you identify and remediate risks that could potentially undermine the confidentiality, integrity, and availability of your cloud data.
+
+This integration currently supports Amazon Web Services (AWS). Support for other major cloud service providers is coming soon. For a step-by-step getting started guide, refer to <<cspm-get-started,Get started with CSPM>>.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm.asciidoc:6]: Just putting it out there: we typically avoid forward-looking statements that seem to promise future features or support, for best practices and maybe also legal reasons. There are also several similar statements in the FAQs, maybe elsewhere.

File: docs/cloud-native-security/cspm.asciidoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm.asciidoc:11]:
```suggestion
* The CSPM integration is available to all Elastic Cloud users. On-premises deployments require an https://www.elastic.co/pricing[Enterprise subscription].
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+== Overview
+
+This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) integration.
+
+NOTE: This integration currently supports Amazon Web Service (AWS). Support for other major cloud service providers is coming soon.
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:15]:
```suggestion
* The CSPM integration is available to all Elastic Cloud users. On-premises deployments require an https://www.elastic.co/pricing[Enterprise subscription].
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+* To view posture data, you need `read` privileges for the following {es} indices: `logs-cloud_security_posture.findings_latest-*`, `logs-cloud_security_posture.scores-*`, `Logs-cloud_security_posture.findings`.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:16]: Thinking out loud: would the list of indices work as a bulleted list (sub-bullets)? It's just a little hard to parse visually as a sentence, commas blend in with other monospace characters.

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+* The user who gives the CSPM integration AWS permissions needs to be an AWS account `admin` user.
+--
+
+[discrete]
+[[cspm-setup]]
+== Set up CSPM for AWS
+
+[discrete]
+[[cspm-add-and-name-integration]]
+=== Add and name your CSPM integration
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:26]: Creating a name doesn't seem like such a central or unique part of this process that it needs to be mentioned in the heading; just "add the integration" seems enough to capture the overall step in the process.
```suggestion
=== Add your CSPM integration
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+* From the Elastic Security homepage, click *Add integrations*.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:27]: I don't think there's an "Elastic Security homepage" or a page called "Home." Maybe "Get started"?

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+* Give your integration a name that matches the purpose or team of the AWS account you want to monitor, for example, `dev-aws-account`.
+
+[discrete]
+=== Set up cloud account access
+The CSPM integration requires access to AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy] in order to discover and evaluate resources in your cloud account. There are several ways to provide access:
+
+* IAM Role (recommended)
+* Direct access keys
+* Temporary security credentials
+* Shared credentials file
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:39]: Might be useful to add links to the sections below.

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+
+[discrete]
+[[cspm-use-irsa]]
+==== Option 1 - [Recommended] Use Kubernetes Service Account to assume IAM role
+
+Leverage IRSA (IAM role to Kubernetes service-account) feature to get temporary credentials and scoped permission.
+From https://aws.github.io/aws-eks-best-practices/security/docs/iam/#iam-roles-for-service-accounts-irsa[EKS Best Practices]:
+
+IRSA is a feature that allows you to assign an IAM role to a Kubernetes service account. It works by leveraging a Kubernetes feature known as Service Account Token Volume Projection. When pods are configured with a Service Account that references an IAM Role, the Kubernetes API server will call the public OIDC discovery endpoint for the cluster on startup. The endpoint cryptographically signs the OIDC token issued by Kubernetes and the resulting token is mounted as a volume. This signed token allows the pod to call the AWS API's associated IAM role. When an AWS API is invoked, the AWS SDK calls `sts:AssumeRoleWithWebIdentity`. After validating the token's signature, IAM exchanges the Kubernetes issued token for a temporary AWS role credential.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:50]: Whoa, we do NOT want to be copy/pasting from AWS's docs. Links and summaries are OK, but quoting another vendor's content could be problematic.

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+* Temporary security credentials
+* Shared credentials file
+
+Regardless of which option you use, you’ll need to attach AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy].
+
+[discrete]
+[[cspm-use-irsa]]
+==== Option 1 - [Recommended] Use Kubernetes Service Account to assume IAM role
+
+Leverage IRSA (IAM role to Kubernetes service-account) feature to get temporary credentials and scoped permission.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:47]:
```suggestion
Leverage the IRSA (IAM role to Kubernetes service-account) feature to get temporary credentials and scoped permission.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+
+To use this, do not fill in any option in the "Setup Access" section. Click *Save and continue*, and follow the https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html[IRSA] guide.
+
+[discrete]
+[[cspm-use-instance-role]]
+==== Option 2 - Use default instance role
+From https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html[IAM roles for Amazon EC2]:
+
+When you create an IAM role using the IAM console, the console creates an instance profile automatically and gives it the same name as the role to which it corresponds.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:59]: Same as above on quoting AWS's docs.

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+
+To use this, do not fill in any option in the "Setup Access" section. Click *Save and continue*.
+
+[discrete]
+[[cspm-use-keys-directly]]
+==== Option 3 - Use access keys directly
+Access keys are long-term credentials for an IAM user or the AWS account root user. To use access keys as credentials, you need to provide the `Access key ID` and the `Secret Access Key`.
+
+For more details, refer to https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html[Access Keys and Secret Access Keys].
+IMPORTANT: You must select "Programmatic access" when creating the IAM user.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:69]: Need an extra blank line for this to render correctly as an AsciiDoc admonition:
```suggestion

IMPORTANT: You must select "Programmatic access" when creating the IAM user.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+
+[discrete]
+[[cspm-use-temp-credentials]]
+==== Option 4 - Use temporary security credentials
+You can configure temporary security credentials in AWS to last for a specified duration. They consist of an access key ID, a secret access key, and a security token, which is typically found using `GetSessionToken`.
+
+Because temporary security credentials are short term, once they expire, you will need to generate new ones and manually update the integration's configuration to continue collecting cloud posture data. Update the credentials before they expire to avoid data loss.
+
+NOTE: IAM users with multi-factor authentication (MFA) enabled need to submit an MFA code when calling `GetSessionToken`. For more details, refer to AWS' https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[Temporary Security Credentials] documentation.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:78]:
```suggestion
NOTE: IAM users with multi-factor authentication (MFA) enabled need to submit an MFA code when calling `GetSessionToken`. For more details, refer to AWS's https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html[Temporary Security Credentials] documentation.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+
+You can use the AWS CLI to generate temporary credentials. For example, you could use the following command if you have MFA enabled:
+
+[source,console]
+----------------------------------
+`sts get-session-token --serial-number arn:aws:iam::1234:mfa/your-email@example.com --duration-seconds 129600 --token-code 123456`
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:84]: I think the backticks are redundant here, because the `source` block is already rendering the line as code

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+The output from this command includes the following fields, which you should provide when configuring the KSPM integration:
+
+* `Access key ID`: The first part of the access key.
+* `Secret Access Key`: The second part of the access key.
+* `Session Token`: A token required when using temporary security credentials.
+
+[discrete]
+[[cspm-use-a-shared-credentials-file]]
+==== Option 5 - Use a shared credentials file
+If you use different AWS credentials for different tools or applications, you can use profiles to define multiple access keys in the same configuration file. For more details refer to AWS' https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html[Shared Credentials Files] documentation.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:96]:
```suggestion
If you use different AWS credentials for different tools or applications, you can use profiles to define multiple access keys in the same configuration file. For more details refer to AWS's https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html[Shared Credentials Files] documentation.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+An IAM role Amazon Resource Name (ARN) is an IAM identity that you can create in your AWS account. You define the role's permissions.
+Roles do not have standard long-term credentials such as passwords or access keys.
+Instead, when you assume a role, it provides you with temporary security credentials for your session.
+
+[discrete]
+[[cspm-finish-setup]]
+=== Finish CSPM setup
+Once you’ve provided AWS credentials:
+
+* If you want to monitor an AWS account where you have not yet deployed {agent}, select *New Hosts* under “Where to add this integration”.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:122]: I think both would be bold, since they're both referring to elements in the UI the user is interacting with.
```suggestion
* If you want to monitor an AWS account where you have not yet deployed {agent}, select *New Hosts* under *Where to add this integration*.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+* Name the {agent} policy. Use a name that matches the purpose or team of the cloud account or accounts you want to monitor. For example, `dev-aws-account`.
+* Click *Save and continue*, then *Add {agent} to your hosts*. The Add agent wizard appears and provides {agent} binaries which you can download and deploy to your AWS account.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:124]:
```suggestion
* Click *Save and continue*, then *Add {agent} to your hosts*. The *Add agent* wizard appears and provides {agent} binaries which you can download and deploy to your AWS account.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+Roles do not have standard long-term credentials such as passwords or access keys.
+Instead, when you assume a role, it provides you with temporary security credentials for your session.
+
+[discrete]
+[[cspm-finish-setup]]
+=== Finish CSPM setup
+Once you’ve provided AWS credentials:
+
+* If you want to monitor an AWS account where you have not yet deployed {agent}, select *New Hosts* under “Where to add this integration”.
+* Name the {agent} policy. Use a name that matches the purpose or team of the cloud account or accounts you want to monitor. For example, `dev-aws-account`.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started.asciidoc:123]: Looks like we're suggesting the customer use the same name for both their Agent policy and their integration policy (above, line 30). That seems potentially confusing, but I don't know enough about cloud use cases so maybe it's not?

**Review by joepeeples — COMMENTED** [review]
Great work wrangling all this new content across multiple files and restructuring the TOC! 😅 The two major areas I'd recommend considering are the forward-looking statements ("X feature will be available in a future version") and quoted chunks from AWS's docs. Maybe check with @jmikell821 about how as a team we should handle those kinds of situations.

File: docs/cloud-native-security/cspm-get-started.asciidoc (hunk 17 above)
Comment (benironside) [reply docs/cloud-native-security/cspm-get-started.asciidoc:123]: I'll refer this question to @tinnytintin10 for his review

File: docs/cloud-native-security/get-started-with-kspm.asciidoc [hunk 18]
```diff
@@ -123,27 +123,22 @@ If you are using the AWS visual editor to create and modify your IAM Policies, y
 
 [discrete]
 [[kspm-use-irsa]]
-===== Option 1 - [Recommended] Use Kubernetes Service Account to assume IAM role
+==== Option 1 - [Recommended] Use Kubernetes Service Account to assume IAM role
 
-Leverage IRSA (IAM role to Kubernetes service-account) feature to get temporary credentials and scoped permission.
-From https://aws.github.io/aws-eks-best-practices/security/docs/iam/#iam-roles-for-service-accounts-irsa[EKS Best Practices]:
+Follow AWS's https://aws.github.io/aws-eks-best-practices/security/docs/iam/#iam-roles-for-service-accounts-irsa[EKS Best Practices] documentation to use the https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html[IRSA][IAM Role to Kubernetes Service-Account] (IRSA) feature to get temporary credentials and scoped permissions.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/get-started-with-kspm.asciidoc:128]:
```suggestion
Follow AWS's https://aws.github.io/aws-eks-best-practices/security/docs/iam/#iam-roles-for-service-accounts-irsa[EKS Best Practices] documentation to use the https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.htm[IAM Role to Kubernetes Service-Account (IRSA)] feature to get temporary credentials and scoped permissions.
```

File: docs/cloud-native-security/security-posture-management.asciidoc [hunk 19]
```diff
@@ -0,0 +1,44 @@
+[[security-posture-management]]
+= Security posture management
+
+[discrete]
+== Overview
+The <<cspm,Cloud Security Posture Management>> (CSPM) and <<kspm,Kubernetes Security Posture Management>> (KSPM) integrations discover and evaluate the services and resources in your cloud environment — like storage, compute, IAM, and more — against security guidelines defined by the Center for Internet Security (CIS). They help you identify and remediate configuration risks that could undermine the confidentiality, integrity, and availability of your cloud assets, such as publicly exposed storage buckets or overly permissive networking objects.
+
+The KSPM integration can assess the security of your Kubernetes assets, while the CSPM integration assesses the security of your AWS resources such as storage, compute, IAM, and more.
+
+[discrete]
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/security-posture-management.asciidoc:10]: Need an anchor here :)

File: docs/cloud-native-security/security-posture-management.asciidoc [hunk 20]
```diff
@@ -0,0 +1,44 @@
...
+. Click *View all failed findings*, either for an individual resource or a group of resources.
+. Click a failed finding to open the Findings flyout.
+. Follow the steps under Remediation to fix the misconfiguration.
+
+*Identify the CIS Sections (security best practice categories) with which your resources are least compliant:*
+
+. Go to the Cloud Posture dashboard (*Dashboards > Cloud Posture*).
+. Under Failed findings by CIS section, click the name of a CIS section to view all failed findings from that section.
+
+Alternatively:
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/security-posture-management.asciidoc:35]: This sticks out a bit. What do you think about combining this with step 2? Maybe something like:

2. Choose one of the following:
 - Under Failed findings by CIS section, click the name of a CIS section to view all failed findings from that section.
 - Go to the Findings page and filter by the rule.section field. For example, search for rule.section : API Server to view findings from the API Server category.

File: docs/dashboards/cloud-posture.asciidoc [hunk 21]
```diff
@@ -2,16 +2,16 @@
 // Note: This page is intentionally duplicated by docs/cloud-native-security/cloud-nat-sec-posture.asciidoc. When you update this page, update that page to match. And careful with the anchor links because they should not match.
 
 = Cloud Posture dashboard
-The Cloud Posture dashboard summarizes your Kubernetes infrastructure's overall performance against CIS <<benchmark-rules,security benchmarks>>. To learn how to collect this data from your clusters, refer to <<get-started-with-kspm, Get started with KSPM>>.
+The Cloud Posture dashboard summarizes your cloud infrastructure's overall performance against CIS <<benchmark-rules,security guidelines>> defined by the Center for Internet Security (CIS). To get started monitoring your security posture, refer to <<cspm-get-started, Get started with Cloud Security Posture Management>> or <<get-started-with-kspm, Get started with Kubernetes Security Posture Management>>.
```
Comment (nastasha-solomon) [inline docs/dashboards/cloud-posture.asciidoc:5]:
```suggestion
The Cloud Posture dashboard summarizes your cloud infrastructure's overall performance against CIS <<benchmark-rules,security guidelines>>, defined by the Center for Internet Security (CIS). To get started monitoring your security posture, refer to <<cspm-get-started, Get started with Cloud Security Posture Management>> or <<get-started-with-kspm, Get started with Kubernetes Security Posture Management>>.
```

**Review by nastasha-solomon — COMMENTED** [review]
Huge effort here, good job! I left some suggestions and questions for you, but overall, everything looks great!

File: docs/cloud-native-security/cloud-nat-sec-posture-dashboard.asciidoc [hunk 22]
```diff
@@ -22,20 +22,22 @@ The Cloud Posture dashboard shows:
...
-The top section of the dashboard summarizes your overall cloud security posture (CSP) by aggregating data from all monitored Kubernetes clusters. Three summary cards on the left of this section show Clusters Evaluated, Resources Evaluated, and Failed Findings. You can click *Resources Evaluated* or *Failed Findings* to open the <<findings-page, Findings page>> grouped by resources or filtered for failed findings, respectively.
+At the top of the dashboard, you can switch between the Cloud accounts and Kubernetes cluster views.
 
-The remaining summary cards show your overall compliance score, and total failed findings grouped by CIS section. Click any CIS section name to view failed findings from that section on the <<findings-page, Findings page>>.
+The top section of either view summarizes your overall cloud security posture (CSP) by aggregating data from all monitored resources. Three summary cards on the left of this section show Cloud Accounts or Clusters evaluated, Resources evaluated, and Failed Findings. You can click *Resources Evaluated* or *Failed Findings* to open the <<findings-page, Findings page>> grouped by resources or filtered for failed findings, respectively.
```
Comment (jmikell821) [inline docs/cloud-native-security/cloud-nat-sec-posture-dashboard.asciidoc:29]:
```suggestion
The top section of either view summarizes your overall cloud security posture (CSP) by aggregating data from all monitored resources. Three summary cards on the left show Cloud Accounts or Clusters evaluated, Resources evaluated, and Failed Findings. You can click *Resources Evaluated* or *Failed Findings* to open the <<findings-page, Findings page>>, grouped by resources or filtered for failed findings, respectively.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 23]
```diff
@@ -0,0 +1,124 @@
...
+--
+
+[discrete]
+[[cspm-setup]]
+== Set up CSPM for AWS
+
+[discrete]
+[[cspm-add-and-name-integration]]
+=== Add your CSPM integration
+* From the Elastic Security *Get started* page, click *Add integrations*.
```
[the remark this answers is not in the document]
Comment (jmikell821) [reply docs/cloud-native-security/cspm-get-started.asciidoc:30]: Yes, definitely an ordered list.

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 24]
```diff
@@ -0,0 +1,124 @@
...
+
+Regardless of which option you use, you’ll need to attach AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy].
+
+[discrete]
+[[cspm-use-irsa]]
+==== Option 1 - [Recommended] Use Kubernetes Service Account to assume IAM role
+
+Follow AWS's https://aws.github.io/aws-eks-best-practices/security/docs/iam/#iam-roles-for-service-accounts-irsa[EKS Best Practices] documentation to use the https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html[IRSA][IAM Role to Kubernetes Service-Account] (IRSA) feature to get temporary credentials and scoped permissions.
+
+During setup, do not fill in any option in the "Setup Access" section. Instead click *Save and continue*.
```
Comment (jmikell821) [inline docs/cloud-native-security/cspm-get-started.asciidoc:53]:
```suggestion
During setup, do not fill in any option in the "Setup Access" section. Click *Save and continue*.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 25]
```diff
@@ -0,0 +1,124 @@
...
+During setup, do not fill in any option in the "Setup Access" section. Click *Save and continue*.
+
+[discrete]
+[[cspm-use-keys-directly]]
+==== Option 3 - Use access keys directly
+Access keys are long-term credentials for an IAM user or the AWS account root user. To use access keys as credentials, you need to provide the `Access key ID` and the `Secret Access Key`.
+
+For more details, refer to https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html[Access Keys and Secret Access Keys].
+
+IMPORTANT: You must select "Programmatic access" when creating the IAM user.
```
Comment (jmikell821) [inline docs/cloud-native-security/cspm-get-started.asciidoc:69]:
```suggestion
IMPORTANT: You must select *Programmatic access* when creating the IAM user.
```

File: docs/cloud-native-security/get-started-with-kspm.asciidoc [hunk 26]
```diff
@@ -123,27 +123,22 @@ If you are using the AWS visual editor to create and modify your IAM Policies, y
...
+Follow AWS's https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html[IAM roles for Amazon EC2] documentation to create an IAM role using the IAM console, which automatically generates an instance profile.
 
-To use this, do not fill in any part of the "Setup Access" section. Click *Save and continue*.
+During setup, do not fill in any option in the "Setup Access" section. Click *Save and continue*.
 
 [discrete]
 [[kspm-use-keys-directly]]
-===== Option 3 - Use access keys directly
+==== Option 3 - Use access keys directly
 Access keys are long-term credentials for an IAM user or the AWS account root user. To use access keys as credentials, you need to provide the `Access key ID` and the `Secret Access Key`.
```
Comment (jmikell821) [inline docs/cloud-native-security/get-started-with-kspm.asciidoc:142]:
```suggestion
Access keys are long-term credentials for an IAM user or AWS account root user. To use access keys as credentials, you must provide the `Access key ID` and the `Secret Access Key`.
```

File: docs/cloud-native-security/get-started-with-kspm.asciidoc [hunk 27]
```diff
@@ -174,7 +169,7 @@ The output from this command includes the following fields, which you should pro
 
 [discrete]
 [[kspm-use-a-shared-credentials-file]]
-===== Option 5 - Use a shared credentials file
+==== Option 5 - Use a shared credentials file
 If you use different AWS credentials for different tools or applications, you can use profiles to define multiple access keys in the same configuration file. For more details refer to AWS' https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html[Shared Credentials Files] documentation.
```
Comment (jmikell821) [inline docs/cloud-native-security/get-started-with-kspm.asciidoc:173]:
```suggestion
If you use different AWS credentials for different tools or applications, you can use profiles to define multiple access keys in the same configuration file. For more details, refer to AWS' https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html[Shared Credentials Files] documentation.
```

File: docs/cloud-native-security/get-started-with-kspm.asciidoc [hunk 28]
```diff
@@ -191,7 +186,7 @@ If you don't provide values for all configuration fields, the integration will u
 
 [discrete]
 [[kspm-use-iam-arn]]
-===== Option 6 - Use an IAM role Amazon Resource Name (ARN)
+==== Option 6 - Use an IAM role Amazon Resource Name (ARN)
 An IAM role Amazon Resource Name (ARN) is an IAM identity that you can create in your AWS account. You define the role's permissions.
 Roles do not have standard long-term credentials such as passwords or access keys.
 Instead, when you assume a role, it provides you with temporary security credentials for your session.
```
Comment (jmikell821) [inline docs/cloud-native-security/get-started-with-kspm.asciidoc:192]:
```suggestion
Instead, when you assume a role, it provides temporary security credentials for your session.
```

File: docs/cloud-native-security/faq.asciidoc [hunk 29]
```diff
@@ -1,15 +1,49 @@
...
+
+*Can I onboard multiple AWS accounts for posture monitoring?*
+
+Yes, you can monitor more than one cloud account. <<cspm-get-started, Add the CSPM integration>> again for each additional account you want to monitor.
+
+*Can I onboard multiple accounts via AWS organizations?*
+
+This capability is not currently supported but will be added in future iterations.
+
+*When do newly-enrolled cloud accounts appear on the dashboard?*
```
Comment (jmikell821) [inline docs/cloud-native-security/faq.asciidoc:25]: Adverbs usually don't required a hyphen after. 
```suggestion
*When do newly enrolled cloud accounts appear on the dashboard?*
```

File: docs/cloud-native-security/faq.asciidoc [hunk 30]
```diff
@@ -1,15 +1,49 @@
...
+
+After you deploy the CSPM integration, it can take up to 10 minutes for resource fetching, evaluation, and data processing before a newly-enrolled account appears on the Cloud Posture dashboard.
```
Comment (jmikell821) [inline docs/cloud-native-security/faq.asciidoc:27]:
```suggestion
After you deploy the CSPM integration, it can take up to 10 minutes for resource fetching, evaluation, and data processing before a newly enrolled account appears on the Cloud Posture dashboard.
```

**Review by jmikell821 — COMMENTED** [review]
Hey @benironside, as with these larger PRs, comments, suggestions, added/deleted lines, etc. can make it difficult to view changes as a whole. Let's get these comments from Nastasha and I incorporated, then I'll look again with fresh eyes. Thanks!

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 31]
```diff
@@ -0,0 +1,124 @@
...
+
+Instead of providing the `Access key ID` and `Secret Access Key` to the integration, provide the information required to locate the access keys within the shared credentials file:
+
+* `Credential Profile Name`: The profile name in the shared credentials file.
+* `Shared Credential File`: The directory of the shared credentials file.
+
+If you don't provide values for all configuration fields, the integration will use these defaults:
+
+- If `Access key ID`, `Secret Access Key`, and `ARN Role` are not provided, then the integration will check for `Credential Profile Name`.
+- If there is no `Credential Profile Name`, the default profile will be used.
```
[the remark this answers is not in the document]
Comment (benironside) [reply docs/cloud-native-security/cspm-get-started.asciidoc:110]: I'm not sure quite what you're picturing here, but if you've got an improvement to the organization in mind I'd love to see it!

File: docs/cloud-native-security/security-posture-management.asciidoc (hunk 20 above)
Comment (benironside) [reply docs/cloud-native-security/security-posture-management.asciidoc:35]: 💯 💯

File: docs/cloud-native-security/cspm-get-started.asciidoc (hunk 31 above)
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/cloud-native-security/cspm-get-started.asciidoc:111]: Yeah, ofc! I was envisioning the following:

```suggestion
- If Access key ID, Secret Access Key, and ARN Role are not provided, then the integration will check for Credential Profile Name. If there is no `Credential Profile Name`, the default profile will be used.
```

It's a lot of info to load into the first bullet but if bullets one and two are related, maybe it's best to keep them together? If the second bullet isn't dependent on the first (i.e., the `Credential Profile Name` can be generated or set in lieu of the user providing the `Access key ID`, `Secret Access Key`, and `ARN Role` are not provided, then the integration will check for `Credential Profile Name`), the list works as is. 

All this said, I don't have a preference towards one or the other. :) Even if the second bullet is dependent on the first, separating out the list this way prevents information overload in the first bullet and creates a more readable list.

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 32]
```diff
@@ -0,0 +1,129 @@
+[[cspm-get-started]]
+= Get started with CSPM
+
+[discrete]
+[[cspm-overview]]
+== Overview
+
+This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) integration.
```
Comment (tinnytintin10) [inline docs/cloud-native-security/cspm-get-started.asciidoc:8]:
```suggestion
This page explains how to get started monitoring the security posture of your cloud assets using the Cloud Security Posture Management (CSPM) feature.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 33]
```diff
@@ -0,0 +1,129 @@
...
+
+NOTE: This integration currently supports Amazon Web Service (AWS).
```
Comment (tinnytintin10) [inline docs/cloud-native-security/cspm-get-started.asciidoc:10]:
```suggestion
NOTE: The CSPM feature currently only supports posture evaluations for Amazon Web Service (AWS).
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 34]
```diff
@@ -0,0 +1,129 @@
...
+** `logs-cloud_security_posture.scores-*`
+** `Logs-cloud_security_posture.findings`
+* The user who gives the CSPM integration AWS permissions needs to be an AWS account `admin` user.
+--
+
+[discrete]
+[[cspm-setup]]
+== Set up CSPM for AWS
+
+To set up CSPM for AWS, add your CSPM integration and then enable cloud account access.
```
Comment (tinnytintin10) [inline docs/cloud-native-security/cspm-get-started.asciidoc:27]:
```suggestion
To set up CSPM for AWS, add the CSPM integration and then enable cloud account access.
```

File: docs/cloud-native-security/cspm-get-started.asciidoc [hunk 35]
```diff
@@ -0,0 +1,129 @@
...
+* <<cspm-use-keys-directly,Direct access keys>>
+* <<cspm-use-temp-credentials,Temporary security credentials>>
+* <<cspm-use-a-shared-credentials-file,Shared credentials file>>
+* <<cspm-use-iam-arn, IAM role Amazon Resource Name (ARN)>>
+
+Regardless of which option you use, you’ll need to attach AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy].
+
+[discrete]
+[[cspm-use-irsa]]
+==== Option 1 - [Recommended] Use Kubernetes Service Account to assume IAM role
```
Comment (tinnytintin10) [inline docs/cloud-native-security/cspm-get-started.asciidoc:54]: Get rid of option 1. This option doesn't apply to CSPM

File: docs/cloud-native-security/cloud-nat-sec-posture-dashboard.asciidoc [hunk 36]
```diff
@@ -1,18 +1,18 @@
...
 // Note: This page is intentionally duplicated by docs/dashboards/cloud-posture.asciidoc. When you update this page, update that page to match. And careful with the anchor links because they should not match.
 
-== Cloud Posture dashboard
+= Cloud Posture dashboard
 
-The Cloud Posture dashboard summarizes your Kubernetes infrastructure's overall performance against CIS <<benchmark-rules,security benchmarks>>. To learn how to collect this data from your clusters, refer to <<get-started-with-kspm, Get started with KSPM>>.
+The Cloud Posture dashboard summarizes your cloud infrastructure's overall performance against <<benchmark-rules,security guidelines>> defined by the Center for Internet Security (CIS). To get started monitoring your security posture, refer to <<cspm-get-started, Get started with Cloud Security Posture Management>> or <<get-started-with-kspm, Get started with Kubernetes Security Posture Management>>.
 
 [role="screenshot"]
 image::images/cloud-sec-dashboard.png[The cloud Security dashboard]
```
Comment (tinnytintin10) [inline docs/cloud-native-security/cloud-nat-sec-posture-dashboard.asciidoc:9]: Screenshot 2023-03-29 at 13 46 05

Let's use the screenshot above instead ^

**Review by tinnytintin10 — CHANGES_REQUESTED** [review]
@benironside I have reviewed and provided my suggestions. Thanks for the work on this!

**Comment by tinnytintin10** [comment]
@olegsu can you pls review the getting started with CSPM doc for technical accuracy? Thanks!

cc @ari-aviran
